dotnetv3/StepFunctions/Actions/StepFunctionsWrapper.cs

```diff
@@ -0,0 +1,283 @@
+    {
+        var response = await _amazonStepFunctions.CreateActivityAsync(new CreateActivityRequest { Name = activityName });
+        return response.ActivityArn;
+    }
+
+    // snippet-end:[StepFunctions.dotnetv3.CreateActivity]
+
+    // snippet-start:[StepFunctions.dotnetv3.CreateStateMachine]
+    /// <summary>
+    /// Create a Step Functions state machine.
+    /// </summary>
+    /// <param name="stateMachineName">Name for the new Step Functions state
+    /// machine.</param>
+    /// <param name="definition">A JSON string that defines the Step Functions
+    /// state machine.</param>
+    /// <param name="roleArn">The Amazon Resource Name (ARN) of the role.</param>
+    /// <returns></returns>
+    public async Task<string> CreateStateMachine(string stateMachineName, string definition, string roleArn)
+    {
+        var request = new CreateStateMachineRequest
+        {
+            Name = stateMachineName,
+            Definition = definition,
+            RoleArn = roleArn
+        };
+
+        var response =
+            await _amazonStepFunctions.CreateStateMachineAsync(request);
+        return response.StateMachineArn;
+    }
+
+    // snippet-end:[StepFunctions.dotnetv3.CreateStateMachine]
+
+    // snippet-start:[StepFunctions.dotnetv3.DeleteActivity]
+    /// <summary>
+    /// Delete a Step Machine activity.
+    /// </summary>
+    /// <param name="activityArn">The Amazon Resource Name (ARN) of
+    /// the activity.</param>
+    /// <returns>A Boolean value indicating the success of the action.</returns>
+    public async Task<bool> DeleteActivity(string activityArn)
+    {
+        var response = await _amazonStepFunctions.DeleteActivityAsync(new DeleteActivityRequest { ActivityArn = activityArn });
+        return response.HttpStatusCode == HttpStatusCode.OK;
+    }
+
+    // snippet-end:[StepFunctions.dotnetv3.DeleteActivity]
+
+    // snippet-start:[StepFunctions.dotnetv3.DeleteStateMachine]
+    /// <summary>
+    /// Delete a Step Functions state machine.
+    /// </summary>
+    /// <param name="stateMachineArn">The Amazon Resource Name (ARN) of the
+    /// state machine.</param>
+    /// <returns>A Boolean value indicating the success of the action.</returns>
+    public async Task<bool> DeleteStateMachine(string stateMachineArn)
+    {
+        var response = await _amazonStepFunctions.DeleteStateMachineAsync(new DeleteStateMachineRequest
+        { StateMachineArn = stateMachineArn });
+        return response.HttpStatusCode == HttpStatusCode.OK;
+    }
+
+    // snippet-end:[StepFunctions.dotnetv3.DeleteStateMachine]
+
+    // snippet-start:[StepFunctions.dotnetv3.DescribeExecution]
+    /// <summary>
+    /// Retrieve information about the specified Step Functions execution.
+    /// </summary>
+    /// <param name="executionArn">The Amazon Resource Name (ARN) of the
+    /// Step machines execution.</param>
+    /// <returns>The API response returned by the API.</returns>
```

Comment:
Could not find this method in the tests.

Reply:
Implemented the test code.